packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -57,6 +64,26 @@ export const HelpCenterContactPage: React.FC = () => {
 		} );
 	}, [ isLoading, renderChat.state, renderEmail.render ] );
 
+	const liveChatHeaderText = useMemo( () => {
+		if ( isDefaultLocale( locale ) ) {
+			return __( 'Live chat', __i18n_text_domain__ );
+		}
+
+		return __( 'Live chat (English)', __i18n_text_domain__ );
+	}, [ __, locale ] );
+
+	const emailHeaderText = useMemo( () => {
+		if ( isDefaultLocale( locale ) ) {
+			return __( 'Email', __i18n_text_domain__ );
+		}
+
+		if ( emailSupportLocales.includes( locale ) ) {
+			return `${ __( 'Email', __i18n_text_domain__ ) } (${ getLanguage( locale )?.name })`;
```

Comment:
```suggestion
			const language = getLanguage( locale );
			return language ? `${ __( 'Email', __i18n_text_domain__ ) } (${ language.name })`: __( 'Email', __i18n_text_domain__ );
```

To avoid showing `Email ()`.